packages/client/src/i18n/messages/views/userSetup.ts

```diff
@@ -170,7 +170,7 @@ const messagesToDefine: IUserSetupMessages = {
     id: 'userSetup.waiting'
   },
   assignedOffice: {
-    defaultMessage: 'Assigned office',
+    defaultMessage: 'Assigned registration office',
```

Comment:
I don't see any corresponding PR on niue repo for this. Is it coming ?

Reply:
I gave a PR on niue which is merged by mr. @youshamahmood96